mavis/Mavis.h

```diff
@@ -150,13 +150,18 @@ class Mavis
     typename InstType::PtrType makeInstFromTrace(const TraceInfoType& tinfo, ArgTypes&& ... args)
     {
         return dtrie_->makeInstFromTrace(tinfo, inst_allocator_, std::forward<ArgTypes>(args)...);
+        
     }
 
     template<typename ...ArgTypes>
     typename InstType::PtrType makeInstDirectly(const mavis::ExtractorDirectInfoIF& user_info, ArgTypes&& ... args)
-    {
-        return dtrie_->makeInstDirectly(user_info, inst_allocator_, std::forward<ArgTypes>(args)...);
-    }
+{ 
+    
+        return dtrie_->makeInstDirectly(user_info,inst_allocator_, std::forward<ArgTypes>(args)...);
+        }
+
+    
+    
```

Comment:
Remove extra empty lines added.

Reply:
Also, please put back the space between `user_info` and `inst_allocator`